src/docs/troubleshoot/fixing-common-errors.md

```diff
@@ -61,21 +63,80 @@ at the port you specified. For more information, check out
 **This approach is not recommended**.
 </Banner>
 
-#### Node/Express
+## Some examples for a few frameworks / servers:
```

Comment:
The elements of this section should all share the same heading (`###`) to have a consistent result
So, the following will need to be amended accordingly:
* Node / Express
* Node / Nest
* Python / Gunicorn

Reply:
this was intentional

Reply:
Also curious about this, are you going for "most common language/framework" as the header?

I think having them consistent with h3/`###` makes sense